swagger-spec/nodes/addon_detail.yaml

```diff
@@ -1 +1,57 @@
 # /nodes/{node_id}/addons/{provider}/
+get:
```

Comment:
This endpoint is writeable (for certain addons) and should have a PATCH method definition.